From 6a35d412da0e3dac2735d7eaa5d05c9f67213f9f Mon Sep 17 00:00:00 2001
From: Jeremy Drake <github@jdrake.com>
Date: Wed, 2 Dec 2020 23:07:02 -0800
Subject: [PATCH 3/3] Look in ../share/fritzing from bin dir on mingw.

This seems to be the best option for finding the prefix.
---
 src/utils/folderutils.cpp | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

--- a/src/utils/folderutils.cpp
+++ b/src/utils/folderutils.cpp
@@ -218,7 +218,7 @@
 	}
 
 
-#ifdef Q_OS_WIN
+#if defined(Q_OS_WIN) && !defined(__MINGW32__)
 	m_appPath = QCoreApplication::applicationDirPath();
 #else
 	// look in standard Fritzing location (applicationDirPath and parent folders) then in standard linux locations
@@ -230,6 +230,9 @@
 
 #ifdef Q_OS_MACOS
 		candidates.append(dir.absolutePath() + "/Resources");
+#endif
+#ifdef __MINGW32__
+			candidates.append(QDir::cleanPath(dir.absolutePath() + "/share/fritzing"));
 #endif
 		candidates.append(dir.absolutePath());
 		if (dir.cdUp()) {
